### DISCUSSION TITLE
Peer does not work with prefix while customizing the component switch

### DISCUSSION THREAD

**Discussion by NguyenChiTrung1310** [tailwindlabs/tailwindcss#12177] [Help] [answered]
Hello everyone, could anyone help me to solve this issue?

For example I have a basic HTML (flowbite tailwind: https://flowbite.com/docs/forms/toggle/)

```
<label class="relative inline-flex items-center cursor-pointer">
  <input type="checkbox" value="" class="sr-only peer">
  <div class="w-11 h-6 bg-gray-200 peer-focus:outline-none peer-focus:ring-4 peer-focus:ring-blue-300 dark:peer-focus:ring-blue-800 rounded-full peer dark:bg-gray-700 peer-checked:after:translate-x-full peer-checked:after:border-white after:content-[''] after:absolute after:top-[2px] after:left-[2px] after:bg-white after:border-gray-300 after:border after:rounded-full after:h-5 after:w-5 after:transition-all dark:border-gray-600 peer-checked:bg-blue-600"></div>
  <span class="ml-3 text-sm font-medium text-gray-900 dark:text-gray-300">Toggle me</span>
</label>
```

However my project is using tailwind with prefix `fm-` so it seems doesn't work with `peer`. I mean If I toggle the switch, it doesn't work. 

Please note that the above code is an example code which is not adding prefix `fm-`, I just want to focus on the class `peer`, `peer-checked` and `peer-focus` which are not work with that prefix

**Comment by alex-krasikau** [comment]
Seems to work fine for me - https://play.tailwindcss.com/vEpfIErLIK, would be great if you could send a reproduction on https://play.tailwindcss.com/

**Reply by NguyenChiTrung1310** [reply]
Hi @alex-krasikau , here is a reproduction bug - https://play.tailwindcss.com/I5gf1VFdAb

Update: you should add `prefix: 'fm-'` in the tailwind config

**Reply by NguyenChiTrung1310** [reply]
You didn't add a prefix so your code works well @alex-krasikau

**Reply by wongjn** [reply, marked as answer]
You had an incorrect placement of the `fm-` prefix. There was a class `peer-checked:fm-after:translate-x-full` where you may have meant `peer-checked:after:fm-translate-x-full`. See https://play.tailwindcss.com/DVRWHfExo7

**Reply by NguyenChiTrung1310** [reply]
Thanks @wongjn , I fixed this issue.
